core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/refactoring/WorkspaceImagePathChange.java

```diff
@@ -80,6 +80,16 @@ public class WorkspaceImagePathChange extends Change {
    */
   private IContainer container;
 
+  /**
+   * Whether the ImagePathChange is done on an already copied project
+   */
+  private Boolean postGenerationChange;

```

Comment:
I think you are right. Let me change this.

Reply:
Great. And a documentation on the field would be useful also. I think about sth like: if this field is provided, it is used to determine the name of the project instead of the container's name before the refactoring.